### PR TITLE
Include provider name (in export) when a provider is specified

### PR DESCRIPTION
**Pull request by dczulada**
Also update the reporting parameter templateId

### CODE REVIEW

File: exporter/templates/cat1/r3/_providers.xml [hunk 1]
```diff
@@ -24,6 +24,14 @@
           {{range .Provider.Addresses}}
             {{template "_address.xml" .}}
           {{end}}
+          {{if .Provider.GivenName}}
+            <assignedPerson>
+              <name>
+                <given>{{ .Provider.GivenName}}</given>
+                <family>{{ .Provider.FamilyName}}</family>
```
Comment (ljades) [inline exporter/templates/cat1/r3/_providers.xml]: Any reason for the space between the `{{` and `.Provider.GivenName}}`?

File: exporter/templates/cat1/r3_1/_providers.xml [hunk 2]
```diff
@@ -24,6 +24,14 @@
           {{range .Provider.Addresses}}
             {{template "_address.xml" .}}
           {{end}}
+          {{if .Provider.GivenName}}
+            <assignedPerson>
+              <name>
+                <given>{{ .Provider.GivenName}}</given>
+                <family>{{ .Provider.FamilyName}}</family>
```
Comment (ljades) [inline exporter/templates/cat1/r3_1/_providers.xml]: Same here, it's a super minor styling thing but still noticeable

**Review by ljades — COMMENTED** [review]
It all looks good except for that minor styling thing I noticed.

**Comment by mattrbianchi** [comment]
LGTM, but sadly Louis is right. Go templates will keep that space when it executes which is valid xml but probably not what people will expect.

**Review by mattrbianchi — APPROVED** [review]

**Review by ljades — APPROVED** [review]

File: exporter/templates/cat1/r3/_providers.xml (hunk 1 above)
Comment (ljades) [inline exporter/templates/cat1/r3/_providers.xml]: OK, so I'll just request changes on these spots in that case.

**Review by ljades — CHANGES_REQUESTED** [review]

**Review by ljades — APPROVED** [review]

**Pull request merged**